Specs/NumberWithUnit/Korean/CurrencyModel.json

```diff
@@ -0,0 +1,1903 @@
+[
+  {
+    "Input": "메릴랜드 주 몽고메리 카운티, 7천 5백만 달러 일반 채권, 시리즈 b,   1989년 매뉴팩처러스 하노버 트러스트에 의해 강화된 공공 개선  채권이다. ",
+    "NotSupported": "dotnet",
+    "NotSupportedByDesign": "javascript,python,java",
+    "Results": [
+      {
+        "Text": "7천 5백만",
+        "TypeName": "currency",
+        "Resolution": {
+          "value": "75000000",
+          "unit": "Dollar"
```

Comment:
Are these general "dollar" or a term specific to US dollar? Sorry, I'm not familiar with Korean, so I'm double checking.

Reply:
If all these cases are the general "dollar", can you add at least one case with the term specifically for USD too?

Reply:
These are the general dollar term. The requested will be added with the implementation.